plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLQuery.java

```diff
@@ -209,6 +211,25 @@ private void parseQuery() {
         }
     }
 
+    private boolean isValidSelectItem(@NotNull SelectItem item) {
+        // Workaround for JSQLParser not respecting the `#` comment in MySQL and treating them as valid values

```

Comment:
Maybe we could create an issue for them.

Reply:
There's an issue and they won't fix it. https://github.com/JSQLParser/JSqlParser/issues/454